lib/src/widgets/widget.dart

```diff
@@ -45,11 +45,18 @@ class Dashbook extends StatefulWidget {
   final bool usePreviewSafeArea;
   final GlobalKey<NavigatorState>? navigatorKey;
 
+  final Iterable<LocalizationsDelegate<dynamic>> localizationsDelegates;
+  final Iterable<Locale> supportedLocales;
```

Comment:
Both these variables should be optional as well. We can't force people to use i18n if they don't need to.